### ISSUE TITLE
resolving environment config should work

### ISSUE THREAD

**Issue by gabrielcsapo** [stefanpenner/ember-strict-resolver#22]
@brendenpalmer and I were looking at how to get this running with ember-a11y testing which fails with strict resolver set at the application level.

https://github.com/ember-a11y/ember-a11y-testing/blob/ca0b9165f119c82c4082450eeb99f4797886ca33/addon/instance-initializers/axe-component.js#L43

```
appInstance.resolveRegistration('config:environment');
```

fails to resolve to anything in ember-strict-resolver whereas the ember-resolver/classic will resolve to `(app-name)/config/environment` and use the function _extractDefaultExport to load the file using requirejs and get the default export.

**Comment by gabrielcsapo** [comment]
We found the code that is actually causing this to be the case in ember-resolver/classic https://github.com/ember-cli/ember-resolver/blob/master/addon/resolvers/classic/index.js#L139. In ember-strict resolver we are getting `"(app-name)/configs/environment"` as you can see it hits the default case in ember-strict-resolver where this always returns the pluralized name for the parsed registry call. Thoughts on adding this as a case we handle where we return;

```
else if(type === 'config') {
      moduleName = `${prefix}/config/${name.replace(/\./g, '/')}`;
    }
```

thoughts @stefanpenner @brendenpalmer?
